dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/workflow/WorkflowResourceIntegrationTest.java

```diff
@@ -652,4 +712,559 @@ public void testDeleteScheme() throws Exception{
         workflowResource.deleteScheme(request4, asyncResponse, savedScheme.getId());
     }
 
+    @Test
+    public void Test_Get_Bulk_Actions_For_Query() {
+        final String luceneQuery = "-contentType:Host -baseType:3 +(conhost:48190c8c-42c4-46af-8d1a-0cd5db894797 conhost:SYSTEM_HOST) +languageId:1 +deleted:false +working:true";
+        final HttpServletRequest request = mock(HttpServletRequest.class);
+        final BulkActionForm form = new BulkActionForm(null, luceneQuery);
+        final Response response = workflowResource.getBulkActions(request, form);
+        assertEquals(Response.Status.OK.getStatusCode(), response.getStatus());
+        final ResponseEntityView entityView = ResponseEntityView.class.cast(response.getEntity());
+        final BulkActionView wa = BulkActionView.class.cast(entityView.getEntity());
+        assertNotNull(wa);
+        final List<BulkWorkflowSchemeView> schemes = wa.getSchemes();
+        assertNotNull(schemes);
+
+        final Optional<BulkWorkflowSchemeView> documentManagementOptional = schemes.stream()
+                .filter(bulkWorkflowSchemeView -> DM_WORKFLOW
+                        .equals(bulkWorkflowSchemeView.getScheme().getName())).findFirst();
+        final Optional<BulkWorkflowSchemeView> systemWorkflowOptional = schemes.stream()
+                .filter(bulkWorkflowSchemeView -> SYSTEM_WORKFLOW
+                        .equals(bulkWorkflowSchemeView.getScheme().getName())).findFirst();
+
+        assertTrue(documentManagementOptional.isPresent());
+        assertTrue(systemWorkflowOptional.isPresent());
+
+        final BulkWorkflowSchemeView documentManagementScheme = documentManagementOptional.get();
+        final List<WorkflowAction> documentActions = getAllWorkflowActions(documentManagementScheme);
+
+        assertTrue(documentActions.stream()
+                .anyMatch(action -> SAVE_AS_DRAFT.equals(action.getName())));
+        assertTrue(documentActions.stream()
+                .anyMatch(action -> SEND_FOR_REVIEW.equals(action.getName())));
+        assertTrue(documentActions.stream()
+                .anyMatch(action -> SEND_TO_LEGAL.equals(action.getName())));
+        assertTrue(documentActions.stream().anyMatch(action -> PUBLISH.equals(action.getName())));
+        assertTrue(
+                documentActions.stream().anyMatch(action -> REPUBLISH.equals(action.getName())));
+        assertTrue(
+                documentActions.stream().anyMatch(action -> UNPUBLISH.equals(action.getName())));
+        assertTrue(documentActions.stream().anyMatch(action -> ARCHIVE.equals(action.getName())));
+        assertTrue(documentActions.stream()
+                .anyMatch(action -> TWEET_THIS.equals(action.getName())));
```

Comment:
We can not depend of the Tweeter Action or Actionlet as will be remove it from the starter.

Reply:
removed.